submissions/emmanuelEsuong/htmlForm/htmlForm.html

```diff
@@ -2,45 +2,49 @@
 
 <html lang="en">
 
-
 <head>
     <meta charset="UTF-8">
     <title>Contact Form</title>
 </head>
 
-
 <body>
-    <h1>Contact Us:</h1>
+    
+    <nav><pre>
+        <a href="signupPage.html">SignUp</a>
 
-    <form action="#" method="POST" enctype="multipart/form-data">
+        <a href="../htmlTask/index.html">Product List</a>
 
+        <a href="htmlForm.html">Contact Us</a>
+    </nav></pre>
+    
+    <br>
+    
+    <h1>Contact Us:</h1>
+
+    <form>
         <fieldset>
-            <legend>Contact Form</legend><br> 
-            
+            <legend>Contact Form</legend><br>  
             <!--Full Name-->
                 <label for="fullname">Full Name:</label>  
-                <input type="text" name="fullname" id="fullname" /><br><br>
-
+                <input type="text" name="fullname" id="fullname" />
```

Comment:
Update the naming here @ElJentle to use camelCame or camel-case. The name and id attribute.